src/styles.scss

```
@@ -3,6 +3,7 @@
 @import "bootstrap-icons/font/bootstrap-icons.css";
 @import "bootstrap/scss/bootstrap";
 @import "@danielmoncada/angular-datetime-picker/assets/style/picker.min.css";
+@import "../src/assets/styles/components/index";
```

Previously I added a path alias for `var.scss`, and it should work for the index too.
```suggestion
@import "components/index";
```

---

Fixed